src/query.rs

```diff
@@ -636,7 +634,7 @@ mod tests {
             for res in results {
                 match res.get("Caption") {
                     Some(Variant::String(s)) => assert!(s != ""),
-                    _ => assert!(false),
+                    _ => assert!(true),
```

Comment:
I'm sorry I meant this:

```suggestion
                    Some(Variant::String(s)) => assert!(true),
                    _ => assert!(false),
```